recipes/pysjef/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "pysjef" %}
+{% set version = "1.24.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/molpro/sjef/archive/refs/tags//{{ version }}.tar.gz
+  sha256: 52fbde358acba17d70c4fa2828610352aa4ca05d22a344e167dfc614928d6f18
+
+build:
+  skip: True  # [win]
+  script:
+    - echo __version__ = \"{{ version }}\" > pysjef/pysjef/_version.py  # [unix]
+    - cmake -B cmake-build . -DCMAKE_INSTALL_PREFIX=$PREFIX  # [linux]
+    - CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY" cmake -B cmake-build . -DCMAKE_INSTALL_PREFIX=$PREFIX  # [osx]
+    - cmake --build cmake-build -t install  # [unix]
+    - ctest -j 1 --test-dir cmake-build -E 'Shell.remote.*|test-Command' || ctest -j 1 --rerun-failed --output-on-failure  # [unix]
+    - cd pysjef && {{ PYTHON }} -m pip install . -vv  # [unix]
+    - rm -f $(grep -v /bin/sjef ../cmake-build/install_manifest.txt)  # [unix] 
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.14
+    - make
+  host:
+    - python
+    - pip
+    - boost-cpp >=1.71
```

Comment:
I think you shoudl remove the `>=1.71` here and rely on the conda-forge pinnings instead.

Reply:
Thanks, I've removed the pinning